react/react1/week2/todo-list-app/src/TodoList.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import { todos } from './todos'
+
+const TodoList = () => {
+  //   mark a To-do as done
+  const [todosState, setTodosState] = useState(todos)
```

Comment:
I am confused about why you defined a todoState with list of array?
Either that could be array name or true/false state.
If that is for state of the todo, it should be like useState(false) and names could be better ones, for example: todoChecked, setTodoChecked. Else if that is for array of todos, you can use todos.
.

Reply:
I just didn't understand at all, what I was doing))